documentation/src/test/java/snippets/EventsTest.java

```diff
@@ -23,5 +24,20 @@ public void test() {
                 });
         // end::code[]
 
+        // tag::shortcut[]
+        Multi<String> multi = Multi.createFrom().items("a", "b", "c");
+        multi.invoke(item -> System.out.println("Received item " + item));
+        // end::shortcut[]
+
+        // tag::invoke-uni[]
+        multi.invokeUni(item -> {
```

Comment:
Perhaps a single instruction lambda expression rather than a block?

Reply:
Yes, I try to avoid reference in code displayed in the documentation (as some developers are not used to them), but here, definitely no need for a block.